### PR TITLE
🌐 [i18n-KO] Translated `asr.mdx` to Korean

### PR DESCRIPTION
**Pull request by sim-so**
# What does this PR do?

Translated the `asr.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? 
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)
@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

File: docs/source/ko/tasks/asr.mdx [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+Hugging Face 계정에 로그인하면 모델을 업로드하고 커뮤니티에 공유할 수 있습니다. 토큰을 입력하여 로그인하세요.
+
+```py
+>>> from huggingface_hub import notebook_login
+
+>>> notebook_login()
+```
+
+## Load MInDS-14 dataset[[load-minds-14-dataset]]
```
Comment (sim-so) [inline docs/source/ko/tasks/asr.mdx:52]:
```suggestion
## MInDS-14 데이터 세트 가져오기[[load-minds-14-dataset]]
```

File: docs/source/ko/tasks/asr.mdx [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+
+다음으로 오디오 신호를 처리하기 위한 Wav2Vec2 프로세서를 가져옵니다:
+
+```py
+>>> from transformers import AutoProcessor
+
+>>> processor = AutoProcessor.from_pretrained("facebook/wav2vec2-base")
+```
+
+MInDS-14 데이터 세트의 샘플링 레이트는 8000kHz이므로(이 정보는 [데이터 세트 카드](https://huggingface.co/datasets/PolyAI/minds14)에서 확인할 수 있습니다), 사전 훈련된 Wav2Vec2 모델을 사용하려면 데이터 세트를 16000kHz로 리샘플링해야 합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/asr.mdx:118]:
```suggestion
MInDS-14 데이터 세트의 샘플링 레이트는 8000kHz이므로([데이터 세트 카드](https://huggingface.co/datasets/PolyAI/minds14)에서 확인), 사전 훈련된 Wav2Vec2 모델을 사용하려면 데이터 세트를 16000kHz로 리샘플링해야 합니다:
```

File: docs/source/ko/tasks/asr.mdx [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+
+<Tip>
+
+자동 음성 인식을 위해 모델을 미세 조정하는 더 자세한 예제는 영어 자동 음성 인식을 위한 [블로그 포스트](https://huggingface.co/blog/fine-tune-wav2vec2-english)과 다국어 자동 음성 인식을 위한 [포스트](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2)를 참조하세요.
+
+</Tip>
+
+## 추론하기[[inference]]
+
+좋아요, 이제 모델을 파인튜닝했으니 추론에 사용할 수 있습니다!
```
Comment (sim-so) [inline docs/source/ko/tasks/asr.mdx:313]:
```suggestion
좋아요, 이제 모델을 미세 조정했으니 추론에 사용할 수 있습니다!
```

File: docs/source/ko/tasks/asr.mdx [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+>>> from transformers import pipeline
+
+>>> transcriber = pipeline("automatic-speech-recognition", model="stevhliu/my_awesome_asr_minds_model")
+>>> transcriber(audio_file)
+{'text': 'I WOUD LIKE O SET UP JOINT ACOUNT WTH Y PARTNER'}
+```
+
+<Tip>
+
+텍스트로 변환된 결과가 꽤 괜찮지만 더 좋을 수도 있습니다! 더 나은 결과를 얻으려면 더 많은 예제로 모델을 세밀하게 조정하세요!
```
Comment (sim-so) [inline docs/source/ko/tasks/asr.mdx:338]:
```suggestion
텍스트로 변환된 결과가 꽤 괜찮지만 더 좋을 수도 있습니다! 더 나은 결과를 얻으려면 더 많은 예제로 모델을 미세 조정하세요!
```

File: docs/source/ko/tasks/asr.mdx [hunk 5]
```diff
@@ -34,37 +35,38 @@ The task illustrated in this tutorial is supported by the following model archit
...
+이렇게 하면 전체 데이터 세트에 대한 훈련에 시간을 들이기 전에 모든 것이 작동하는지 실험하고 검증할 수 있습니다.
 
 ```py
 >>> from datasets import load_dataset, Audio
 
 >>> minds = load_dataset("PolyAI/minds14", name="en-US", split="train[:100]")
 ```
 
-Split the dataset's `train` split into a train and test set with the [`~Dataset.train_test_split`] method:
+[`~Dataset.train_test_split`] 메소드를 사용하여 데이터 세트의 `train` 분할을 훈련 세트와 테스트 세트로 나누세요:
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/asr.mdx:63]: [`~Dataset.train_test_split`] 메소드를 사용하여 데이터 세트의 `train` 분할을 훈련 세트와 테스트 세트로 나누세요:
[`~Dataset.train_test_split`] 메소드를 사용하여 데이터 세트의 `train` 을 훈련 세트와 테스트 세트로 나누세요:

-> mind dataset에서 'train' key에 해당하는 데이터를 split하는 의미라서 분할이 빠지는 것이 좋을 것 같습니다.

File: docs/source/ko/tasks/asr.mdx [hunk 6]
```diff
@@ -34,37 +35,38 @@ The task illustrated in this tutorial is supported by the following model archit
...
 
 ```py
 >>> minds = minds.train_test_split(test_size=0.2)
 ```
 
-Then take a look at the dataset:
+그런 다음 데이터 세트를 확인해보세요:
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/asr.mdx:69]: 그런 다음 데이터 세트를 확인해보세요:
그리고 데이터 세트를 확인하세요:
를 추천합니다.

File: docs/source/ko/tasks/asr.mdx [hunk 7]
```diff
@@ -149,15 +151,15 @@ Now create a preprocessing function that:
...
+전체 데이터 세트에 전처리 함수를 적용하려면 🤗 Datasets [`~datasets.Dataset.map`] 함수를 사용하세요. `num_proc` 매개변수를 사용하여 프로세스 수를 늘리면 `map`의 속도를 높일 수 있습니다. [`~datasets.Dataset.remove_columns`] 메소드를 사용하여 필요하지 않은 열을 제거하세요:
 
 ```py
 >>> encoded_minds = minds.map(prepare_dataset, remove_columns=minds.column_names["train"], num_proc=4)
 ```
 
-🤗 Transformers doesn't have a data collator for ASR, so you'll need to adapt the [`DataCollatorWithPadding`] to create a batch of examples. It'll also dynamically pad your text and labels to the length of the longest element in its batch (instead of the entire dataset) so they are a uniform length. While it is possible to pad your text in the `tokenizer` function by setting `padding=True`, dynamic padding is more efficient.
+🤗 Transformers에는 자동 음성 인식용 데이터 콜레이터가 없으므로 예제 배치를 생성하려면 [`DataCollatorWithPadding`]을 조정해야 합니다. 데이터 콜레이터는 이렇게 하면 텍스트와 레이블을 배치에서 가장 긴 요소의 길이에 동적으로 패딩하여 길이를 균일하게 합니다. `tokenizer` 함수에서 `padding=True`를 설정하여 텍스트를 패딩할 수 있지만, 동적 패딩이 더 효율적입니다.
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/asr.mdx:160]: 이렇게 하면 데이터 콜레이터는 텍스트와 레이블을 배치에서 가장 긴 요소의 길이에 동적으로 패딩하여 길이를 균일하게 합니다

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다 👍
사전에 설명해주셔서 개념과 용어를 이해하기 수월하네요! 아래 리뷰를 남겼습니다 😀

File: docs/source/ko/tasks/asr.mdx (hunk 5 above)
Comment (sim-so) [reply docs/source/ko/tasks/asr.mdx:63]:
```suggestion
[`~Dataset.train_test_split`] 메소드를 사용하여 데이터 세트의 `train`을 훈련 세트와 테스트 세트로 나누세요:
```
위 내용으로 수정하겠습니다!

File: docs/source/ko/tasks/asr.mdx (hunk 6 above)
Comment (sim-so) [reply docs/source/ko/tasks/asr.mdx:69]:
```suggestion
그리고 데이터 세트를 확인하세요:
```

File: docs/source/ko/tasks/asr.mdx (hunk 7 above)
Comment (sim-so) [reply docs/source/ko/tasks/asr.mdx:160]:
```suggestion
🤗 Transformers에는 자동 음성 인식용 데이터 콜레이터가 없으므로 예제 배치를 생성하려면 [`DataCollatorWithPadding`]을 조정해야 합니다. 이렇게 하면 데이터 콜레이터는 텍스트와 레이블을 배치에서 가장 긴 요소의 길이에 동적으로 패딩하여 길이를 균일하게 합니다. `tokenizer` 함수에서 `padding=True`를 설정하여 텍스트를 패딩할 수 있지만, 동적 패딩이 더 효율적입니다.
```

File: docs/source/ko/tasks/asr.mdx [hunk 8]
```diff
@@ -300,15 +304,15 @@ Once training is completed, share your model to the Hub with the [`~transformers
 
 <Tip>
 
-For a more in-depth example of how to finetune a model for automatic speech recognition, take a look at this blog [post](https://huggingface.co/blog/fine-tune-wav2vec2-english) for English ASR and this [post](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2) for multilingual ASR.
+자동 음성 인식을 위해 모델을 미세 조정하는 더 자세한 예제는 영어 자동 음성 인식을 위한 [블로그 포스트](https://huggingface.co/blog/fine-tune-wav2vec2-english)과 다국어 자동 음성 인식을 위한 [포스트](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2)를 참조하세요.
```
[the remark this answers is not in the document]
Comment (sim-so) [reply docs/source/ko/tasks/asr.mdx:307]:
```suggestion
자동 음성 인식을 위해 모델을 미세 조정하는 더 자세한 예제는 영어 자동 음성 인식을 위한 [블로그 포스트](https://huggingface.co/blog/fine-tune-wav2vec2-english)와 다국어 자동 음성 인식을 위한 [포스트](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2)를 참조하세요.
```

File: docs/source/ko/tasks/asr.mdx [hunk 9]
```diff
@@ -300,15 +304,15 @@ Once training is completed, share your model to the Hub with the [`~transformers
...
 </Tip>
 
-## Inference
+## 추론하기[[inference]]
 
-Great, now that you've finetuned a model, you can use it for inference!
+좋아요, 이제 모델을 파인튜닝했으니 추론에 사용할 수 있습니다!
 
-Load an audio file you'd like to run inference on. Remember to resample the sampling rate of the audio file to match the sampling rate of the model if you need to!
+추론에 사용할 오디오 파일을 가져오세요. 필요한 경우 오디오 파일의 샘플링 비율을 모델의 샘플링 비율에 맞게 리샘플링하는 것을 잊지 마세요!
```
[the remark this answers is not in the document]
Comment (sim-so) [reply docs/source/ko/tasks/asr.mdx:315]:
```suggestion
추론에 사용할 오디오 파일을 가져오세요. 필요한 경우 오디오 파일의 샘플링 비율을 모델의 샘플링 레이트에 맞게 리샘플링하는 것을 잊지 마세요!
```

**Review by sim-so — COMMENTED** [review]
성모님이 제안해주신 수정 사항을 반영합니다!

**Comment by sim-so** [comment]
Hope you have a great week!
Could you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!
